lib/opensusebasetest.pm

```diff
@@ -336,6 +336,7 @@ sub wait_boot {
             select_console('iucvconn');
         }
         else {
+            workaround_type_encrypted_passphrase if get_var('S390_ZKVM');
```

Comment:
just to confirm , you currently in branch where variable ARCH not equal s390x . and you check for variable S390_ZKVM, right ?

Reply:
No, I'm within the if-branch started in L324, that is ARCH=s390x and in the else-branch to BACKEND=s390x ~ z/VM.

Reply:
ah sorry , mess up with indentation :)